docs/rules/no-is-function.md

```diff
@@ -28,7 +31,10 @@ $div.isFunction();
 🔧 Examples of code **fixed** by this rule:
 ```js
 $.isFunction( expression( arg ) ); /* → */ typeof expression( arg ) === 'function';
-if ( $.isFunction( fn ) ) { g(); } /* → */ if ( typeof fn === 'function' ) { g(); }
```

Comment:
Why aren't these single line anymore? I think it breaks the layout of the fixing examples.